Editor/Resources/EditorWindow/Components/InfoLinks.uss

```diff
@@ -0,0 +1,14 @@
+﻿/* Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved. */
+/* SPDX-License-Identifier: Apache-2.0 */
+
+:root {
+    color: #4C7EFF;
```

Comment:
Can we get this added as a variable in `common.uss`? Something like `--font-color-info-link`

Reply:
moved to common.uss